R/public_api.r

```diff
@@ -59,10 +59,10 @@ list_packages <- function(from = "", limit = 10,
   }
 
   url <- switch(format,
-                "short" = "/-/desc",
-                "latest" = "/-/latest",
-                "full" = "/-/all")
-  if (archived) url <- "/-/allall"
+                "short" = "-/desc",
+                "latest" = "-/latest",
+                "full" = "-/all")
+  if (archived) url <- "-/allall"
 
```

Comment:
I would not remove the leading `/`, just in case the URL is set externally and it does not have a trailing `/`.

Reply:
so how would you handle the fact it should not have the trailing URL? Should I add code in `query()` to remove the second "//" if there's one?

Reply:
No `//` is fine, it will not cause any problems. AFAICT we already created `//` URLs in this package, actually. So just keep the leading slash, and it is all fine.

Reply:
ok!